### PR TITLE
Sp22 reset

### PR DESCRIPTION
**Pull request by laksith19**
Reset website for Spring 22 semester

### CODE REVIEW

File: _data/materials.yaml [hunk 1]
```diff
@@ -2,128 +2,128 @@
...
   advanced-name: Networked Services
   advanced-slides: https://docs.google.com/presentation/d/11nMxy5iJTS3YDBLqT3O5i8cbbRdtS2Hxf4QRbHJiGBo/edit
   advanced-video: https://youtu.be/5clc5xi5FDo
-  beginner-solutions: https://docs.google.com/document/d/1u5KV_LIfUbQHvPbspq_ejHxO1LsxHskMIlcHVgi0AhQ/edit?usp=sharing
-  advanced-solutions: https://docs.google.com/document/d/1YiSOMMNl_H5XNmvEqo53tCQ8xHFurqBMkIq6dWJ7ExE/edit?usp=sharing
-  labdue: Sat. 10/23
-  hidelink: false
+  #beginner-solutions: https://docs.google.com/document/d/1u5KV_LIfUbQHvPbspq_ejHxO1LsxHskMIlcHVgi0AhQ/edit?usp=sharing
+  #advanced-solutions: https://docs.google.com/document/d/1YiSOMMNl_H5XNmvEqo53tCQ8xHFurqBMkIq6dWJ7ExE/edit?usp=sharing
+  labdue: Sat. 3/19
```
Comment (64bitpandas) [inline _data/materials.yaml:92]: maybe we should make this lab due after spring break vs before?

**Review by 64bitpandas — APPROVED** [review]
lgtm! left 1 small comment but everything else is good to go

File: _data/materials.yaml (hunk 1 above)
Comment (lancemathias) [reply _data/materials.yaml:92]: Would be a good idea. Were you thinking Saturday or Sunday after break?

**Review by lancemathias — APPROVED** [review]
Looks good, might want to consider Ben’s comment

File: _data/materials.yaml (hunk 1 above)
Comment (laksith19) [reply _data/materials.yaml:92]: yeah that makes sense, I'll shift it to after the break. Maybe keep it Saturday to be consistent with the other due dates?

**Pull request merged**